### ISSUE TITLE
error in fileupload

### ISSUE THREAD

**Issue by deveix** [EddyVerbruggen/nativescript-plugin-firebase#497]
error: Property 'name' does not exist on type 'UploadFileResult'

please add all the variables inside UploadFileResult because i won't do it myself every time this plugin update

**Comment by EddyVerbruggen** [comment]
Have you considered sending a PR so you do it only once and everybody benefits of your change?

**Comment by caveman86** [comment]
and even UploadFileResult  doesnt have url propriety !
